apis/standard.oam.dev/v1alpha1/rollouttrait_types.go

```diff
@@ -42,6 +42,7 @@ type RolloutTraitSpec struct {
 // RolloutTrait is the Schema for the RolloutTrait API
 // +kubebuilder:object:root=true
 // +genclient
+// +kubebuilder:resource:categories={oam},shortName=rot
```

Comment:
what about `approllout` ? `rot` can hardly be understood

Reply:
`approllout` is the short name of `approllouts`, and `rt` is the short name of `resourcetrackers`. `rollout` stands for `rollouttraits`, `approllout` stands for `approllouts`, what do you think?

Reply:
Oh, sorry. I think this short name is not needed, this trait will be deprecated soon, we will use AppRollout instead.

Reply:
Got it, I have removed it